packages/atclient/include/atclient/version.h

```diff
@@ -1,6 +1,6 @@
 #ifndef ATCLIENT_VERSION_H
 #define ATCLIENT_VERSION_H
 
-#define ATCLIENT_ATSDK_VERSION "0.0.1"
+#define ATCLIENT_ATSDK_VERSION "0.1.0"
```

Comment:
Should this match the version in the change log?

Reply:
Yes... this was a situation where I bumped the version, then went to the releases to check what version we were actually on. Then went to add everything to the change log and forgot to come back.